### PR TITLE
[patch] Chrome SameSite warning - Update package.json

### PR DESCRIPTION
**Pull request by pixnlove**
We need to upgrade https://github.com/jshttp/cookie and https://github.com/expressjs/session in order to be able to set a cookie with SameSite value equal to `none`.
https://www.chromestatus.com/feature/5088147346030592

Capture d’écran 2019-10-07 à 22 57 41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
-    "csurf": "1.9.0",
+    "cookie": "0.4.0",
+    "cookie-parser": "1.4.4",
+    "cookie-signature": "1.1.0",
+    "csurf": "1.10.0",
     "ejs": "2.5.7",
-    "express": "4.16.2",
-    "express-session": "1.15.6",
+    "express": "4.17.1",
+    "express-session": "git+https://github.com/expressjs/session.git",
```
Comment (pixnlove) [inline package.json]: Waiting for express-session 1.16.3 or higher

**Comment by johnabrams7** [comment]
We noticed `express-session` is directly linked to the Express repo.  For safety, we can't merge directly linked dependencies - however I did notice [express-session 1.17.0](https://www.npmjs.com/package/express-session/v/1.17.0) is out if that helps!

File: package.json (hunk 1 above)
Comment (johnabrams7) [reply package.json]: @pixnlove [express-session 1.17.0](https://www.npmjs.com/package/express-session/v/1.17.0) is out 👍

Comment (pixnlove) [reply package.json]: Done @johnabrams7, waiting for review @rachaelshaw

**Comment by kriss1897** [comment]
@johnabrams7 The PR is updated. Can you take a look?

**Comment by svenemtell** [comment]
Waiting eagerly for a new Sails release including this PR.

**Comment by svenemtell** [comment]
Any news on this?
This has become really urgent since the recent Chrome rollout https://www.chromium.org/updates/same-site!
Unfortunately I cannot patch `express-session` locally, since I am running sails on Heroku.

**Review by rachaelshaw — APPROVED** [review]

**Comment by rachaelshaw** [comment]
Merged– thanks @pixnlove for updating this!

**Pull request merged**